src/main/resources/wildfly/configure-wildfly.cli

```diff
@@ -47,7 +47,7 @@ data-source add \
 # Create custom ZAC mail session
 /subsystem=mail/mail-session=zac:add(jndi-name=java:jboss/mail/zac,debug=false)
 /socket-binding-group=standard-sockets/remote-destination-outbound-socket-binding=zac-smtp-binding:add(host=${env.SMTP_SERVER},port=${env.SMTP_PORT})
-/subsystem=mail/mail-session=zac/server=smtp:add(outbound-socket-binding-ref=zac-smtp-binding,username=${env.SMTP_USERNAME},password=${env.SMTP_PASSWORD})
+/subsystem=mail/mail-session=zac/server=smtp:add(outbound-socket-binding-ref=zac-smtp-binding,username=${env.SMTP_USERNAME:∅},password=${env.SMTP_PASSWORD:∅})
```

Comment:
Those `∅` are the default values that we need for things to work? I would definitely add a little comment here as well about this because future developers will certainly wonder about this.

Reply:
And why a `∅`? Does this have special meaning?

Reply:
No special meaning, besides mathematical empty set . Will change these to something more descriptive

Reply:
Using `dummyDefault` prefix for username and password